easybuild/easyconfigs/e/Extrae/Extrae-4.2.0-gompi-2023b.eb

```diff
@@ -32,17 +32,27 @@ builddependencies = [
     ('Automake', '1.16.5'),
 ]
 
-dependencies = [
-    ('zlib', '1.2.13'),
-    ('Boost', '1.83.0'),
-    ('libunwind', '1.6.2'),
-    ('libxml2', '2.11.5'),
-    ('libdwarf', '0.9.2'),
-    ('PAPI', '7.1.0'),
-]
+if ARCH == 'aarch64':
+    dependencies = [
+        ('zlib', '1.2.13'),
+        ('Boost', '1.83.0'),
+        ('libxml2', '2.11.5'),
+        ('libdwarf', '0.9.2'),
+        ('PAPI', '7.1.0'),
+    ]
+    configopts = '--without-unwind'
+else:
+    dependencies = [
+        ('zlib', '1.2.13'),
+        ('Boost', '1.83.0'),
+        ('libunwind', '1.6.2'),
+        ('libxml2', '2.11.5'),
+        ('libdwarf', '0.9.2'),
+        ('PAPI', '7.1.0'),
+    ]
```

Comment:
```suggestion
dependencies = [
    ('zlib', '1.2.13'),
    ('Boost', '1.83.0'),
    ('libunwind', '1.6.2'),
    ('libxml2', '2.11.5'),
    ('libdwarf', '0.9.2'),
    ('PAPI', '7.1.0'),
]

# libunwind causes segv errors on aarch64
if ARCH != 'aarch64':
    dependencies.append(
        ('libunwind', '1.6.2'),
    )
```

Reply:
The relevant config option is auto-injected by the updated easyblock in https://github.com/easybuilders/easybuild-easyblocks/pull/3393